src/azure-cli/azure/cli/command_modules/redis/_params.py

```diff
@@ -60,6 +61,25 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('rule_name', help='The name of the firewall rule.')
         c.argument('start_ip', help='Lowest IP address included in the range.')
 
+    with self.argument_context('redis access-policy') as c:
+        c.argument('access_policy_name', help='The name of the access policy that is being assigned')
+        c.argument('permissions', help='Permissions for the access policy. Learn how to configure permissions at '
+                                       'https://aka.ms/redis/AADPreRequisites')
+
+    with self.argument_context('redis access-policy list') as c:
+        c.argument('cache_name', arg_type=cache_name, id_part=None)
+
+    with self.argument_context('redis access-policy-assignment') as c:
+        c.argument('access_policy_assignment_name', options_list=['--policy-assignment-name'], help='The name of the access '
+                                                                                             'policy assignment')
```

Comment:
```suggestion
        c.argument('access_policy_assignment_name', options_list=['--policy-assignment-name'],
                           help='The name of the access policy assignment')
```
There's a linter check failure for this line